providers/direct/src/main/java/quarks/providers/direct/DirectProvider.java

```diff
@@ -37,6 +37,13 @@
         implements DirectSubmitter<Topology, Job> {
 
     private final ServiceContainer services;
+    
+    /**
+     * CONFIGURATION_JOB_NAME is used to identify the submission configuration property containing the job name.
+     * The value is {@value}.
+     * @see #submit(Topology, JsonObject)
+     */
+    public static final String CONFIGURATION_JOB_NAME = "jobName";
```

Comment:
This doesn't seem like the right location for configuration key values.

Maybe some new interface in the execution api like Configs or Configurations, then the name can simply be JOB_NAME.